# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/client/ClientImageLayer.cpp
# Commit: 96da9d241051
# Full Hash: 96da9d241051d222bdd693bdb519d058a013b59c
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2019-03-23 09:46:24
# Regressor Bug: 1441308
# File Overlap Count: 2
# Description:
#   Bug 1441308 - Core renderroot splitting changes r=kats,sotaro
#   
#   This is a large patch that contains all of the core changes for
#   renderroot splitting.
#   
# ==============================================================================

diff -r 8cefa694f811 -r 96da9d241051 gfx/layers/client/ClientImageLayer.cpp
--- a/gfx/layers/client/ClientImageLayer.cpp	Fri Mar 22 18:28:31 2019 +0000
+++ b/gfx/layers/client/ClientImageLayer.cpp	Fri Mar 22 18:28:42 2019 +0000
@@ -120,8 +120,8 @@
     return;
   }
 
-  if (!mImageClient ||
-      !mImageClient->UpdateImage(mContainer, GetContentFlags())) {
+  if (!mImageClient || !mImageClient->UpdateImage(mContainer, GetContentFlags(),
+                                                  Nothing())) {
     CompositableType type = GetImageClientType();
     if (type == CompositableType::UNKNOWN) {
       return;
@@ -137,7 +137,8 @@
       mImageClient->Connect();
       ClientManager()->AsShadowForwarder()->Attach(mImageClient, this);
     }
-    if (!mImageClient->UpdateImage(mContainer, GetContentFlags())) {
+    if (!mImageClient->UpdateImage(mContainer, GetContentFlags(),
+                                   Nothing())) {
       return;
     }
   }